src/main/kotlin/com/charleskorn/kaml/YamlParser.kt

```diff
@@ -60,23 +59,12 @@ class YamlParser(yamlSource: String) {
 
     private fun checkEvent(retrieve: () -> Event): Event {
         try {
-            val event = retrieve()
-            checkForUnsupportedFeatures(event)
-
-            return event
+            return retrieve()
         } catch (e: MarkedYamlEngineException) {
             throw translateYamlEngineException(e)
         }
     }
 
-    private fun checkForUnsupportedFeatures(event: Event) {
-        if (event is ScalarEvent) {
-            if (event.tag.isPresent) {
-                throw UnsupportedYamlFeatureException("tags", event)
```

Comment:
Given that `UnsupportedYamlFeatureException` won't be thrown any more, could you please remove the class?